skills/consolidate/pr.md

```
@@ -57,6 +57,61 @@ Skip entirely if any of these are true:
 
+## Step 2.4: Copilot availability pre-check (HARD STOP — auto-fallback, no ask)
+
+**GitHub Copilot Code Review is a paid subscription as of 2025.** Without an active subscription on the acting account or the repo's organization, adding `copilot-pull-request-reviewer` as a reviewer fails silently or returns 422. **Check availability BEFORE Step 2.5 sequential trigger or Step 2.6 re-review trigger. On unavailable → automatically route to Internal Review Fallback (Step 3.5) WITHOUT AskUserQuestion.**
+
+### Availability detection (primary-source signals — confirm via at least one)
+
+| # | Signal | Command | Interpretation |
+|---|--------|---------|----------------|
+| 1 | User-level subscription | `GH_TOKEN="$(gh auth token --user <account>)" gh api /user/copilot_billing 2>&1` | 200 with `seat_breakdown` → available. 404 / "Not Found" → **not available** |
+| 2 | Org-level subscription (org repos only) | `GH_TOKEN="$(gh auth token --user <account>)" gh api /orgs/<org>/copilot/billing 2>&1` | 200 with `seat_breakdown.total` > 0 → available for org members. 404 / 403 → **not available** |
+| 3 | Past behavior on this repo | `gh pr list -R <owner>/<repo> --state all --limit 20 --json reviews --jq '[.[].reviews[]? | select(.author.login == "copilot-pull-request-reviewer")] | length'` | `>0` → repo has historically used Copilot review (likely available). `0` → no historical evidence (unknown) |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 4; Actual: 6; Too many cells, extra data will be missing

(MD056, table-column-count)